src/python/WMCore/WMStats/CherryPyThreads/DataCacheUpdate.py

```diff
@@ -17,7 +18,8 @@ def setConcurrentTasks(self, config):
         """
         sets the list of functions which
         """
-        self.concurrentTasks = [{'func': self.gatherActiveDataStats, 'duration': 300}]
+        self.concurrentTasks = [{'func': self.gatherActiveDataStats,
+                                 'duration': config.dataCacheUpdateDuration}]
```

Comment:
Do we have a default value set for `dataCacheUpdateDuration`, either in the config or somewhere in the code?

Reply:
Yes, it's set to 5min.

Reply:
ok